File: ProjectTemplates/DotNetTemplate/MonoGame.Templates.VisualBasic/content/BlankApp/BlankApp.Android/Activity1.vb
Imports Android.App
Imports Android.Content.PM
Imports Android.OS
Imports Android.Views

Namespace BlankApp.Android
    <Activity(Label = "BlankApp.Android",
        MainLauncher = True,
        Icon = "@drawable/icon",
        Theme = "@style/Theme.Splash",
        AlwaysRetainTaskState = True,
        LaunchMode = LaunchMode.SingleInstance,
        ScreenOrientation = ScreenOrientation.FullUser,
        ConfigurationChanges = ConfigChanges.Orientation Or ConfigChanges.Keyboard Or ConfigChanges.KeyboardHidden Or ConfigChanges.ScreenSize)>
    Public Class Activity1 : Inherits Microsoft.Xna.Framework.AndroidGameActivity
        Protected Overrides Sub OnCreate(ByVal bundle As Bundle)
            base.OnCreate(bundle)
            Dim g As New Game1()
            SetContentView(CType(g.Services.GetService(Of View), View))
            g.Run()
        End Sub
    End Class
End Namespace

File: ProjectTemplates/DotNetTemplate/MonoGame.Templates.CSharp/content/BlankApp/BlankApp.Android/Activity1.cs
using Android.App;
using Android.Content.PM;
using Android.OS;
using Android.Views;

namespace BlankApp.Android
{
    [Activity(Label = "BlankApp.Android"
        , MainLauncher = true
        , Icon = "@drawable/icon"
        , Theme = "@style/Theme.Splash"
        , AlwaysRetainTaskState = true
        , LaunchMode = Android.Content.PM.LaunchMode.SingleInstance
        , ScreenOrientation = ScreenOrientation.FullUser
        , ConfigurationChanges = ConfigChanges.Orientation | ConfigChanges.Keyboard | ConfigChanges.KeyboardHidden | ConfigChanges.ScreenSize)]
    public class Activity1 : Microsoft.Xna.Framework.AndroidGameActivity
    {
        protected override void OnCreate(Bundle bundle)
        {
            base.OnCreate(bundle);
            var g = new Game1();
            SetContentView((View)g.Services.GetService(typeof(View)));
            g.Run();
        }
    }
}

